tests/cases/standalone/common/system/timezone.sql

```diff
@@ -0,0 +1,63 @@
+--- tests for timezone ---
+SHOW VARIABLES time_zone;
+
+SHOW VARIABLES system_time_zone;
+
+CREATE TABLE test(d double, ts timestamp_ms time index);
+
+INSERT INTO test values
+       (1, '2024-01-01 00:00:00'),
+       (2, '2024-01-02 08:00:00'),
+       (3, '2024-01-03 16:00:00'),
+       (4, '2024-01-04 00:00:00');
+
+SELECT * from test;
+
+SELECT * from test where ts >= '2024-01-02 08:00:00';
+
+SELECT * from test where ts <= '2024-01-03 16:00:00';
+
+select date_format(ts, '%Y-%m-%d %H:%M:%S:%3f') from test;
+
+select to_unixtime('2024-01-02 00:00:00');
+
+--- UTC+8 ---
+SET TIME_ZONE = '+8:00';
```

Comment:
I just realized PG's timezone is different from other variables: https://www.postgresql.org/docs/current/sql-set.html

```sql
SET TIME ZONE '+08:00';
SET TIME ZONE 'Asia/Shanghai';
```

Maybe we need another PR to align this behavior